internal/sqlchannel/engine/mongodb.go

```diff
@@ -45,8 +45,8 @@ func newMongoDB() *mongodb {
 		examples: map[ClientType]buildConnectExample{
 			CLI: func(info *ConnectionInfo) string {
 				return fmt.Sprintf(`# mongodb client connection example
-mongosh mongodb://%s:%s@%s/%s?replicaset=%s-%s
-`, info.User, info.Password, info.HeadlessEndpoint, info.Database, info.ClusterName, info.ComponentName)
+mongosh mongodb://%s:%s@%s/%s
+`, info.User, info.Password, info.Host, info.Database)
```

Comment:
why replace the mongo url ?

Reply:
see this [issue](https://github.com/apecloud/kubeblocks/issues/4403). mongosh can't connnect this url

Reply:
the dns url should be used in k8s clusters rather than local machine.  host is incorrect.

Reply:
the dns url is just be used in local machine. The dns url that used in k8s clusters is in here, https://github.com/apecloud/kubeblocks/blob/6c71644fa7c1facef7e45857287c1f6c5fb3e261/internal/sqlchannel/engine/mongodb.go#L80-L82C2